test/SoundPlayer.js

```diff
@@ -21,7 +21,7 @@ tap.test('SoundPlayer', suite => {
         audioContext = new AudioContext();
         audioEngine = new AudioEngine(audioContext);
         // sound will be 0.1 seconds long
-        audioContext.DECODE_AUDIO_DATA_RESULT = audioContext.createBuffer(2, 4410, 44100);
+        audioContext.DECODE_AUDIO_DATA_RESULT = audioContext.createBuffer(2, 8820, 44100);
         audioContext.DECODE_AUDIO_DATA_FAILED = false;
         const data = new Uint8Array(44100);
```

Comment:
this can be much smaller, i dunno why i left it this big, the buffer can be like 1024 bytes it doesn't matter cuz the decode result will still be the 8820 above..

Reply:
We just need to pass an item with a buffer (sliceable?) member. You can pass a 0 byte length item.